### ISSUE TITLE
Improve usability of Rescuezilla

### ISSUE THREAD

**Issue by shasheene** [rescuezilla/rescuezilla#6]
There are many aspects to the application's usability that can be improved in future versions of Rescuezilla:

* Redo Backup and Recovery has never had a back button.
* Misclicking on the wrong partition may cause an error box (this is good) but exits the application
* Replace disk selection drop-down menu with a less compact menu (that doesn't require two clicks) (via Sourceforge user [Graham](https://sourceforge.net/p/redobackup/discussion/1169663/thread/6f5a7216/))

While Clonezilla has similar usability issues, that is no excuse.

**Comment by shasheene** [comment]
The CIFS/SMB and FTP navigation leaves much to be desired:

* If a mount takes a long time, the GUI freezes
* There is no clear feedback when a mount fails.

To help troubleshoot, ideally there would be a small informational dialogue that shows something like this:
```
Pinging server... ✔
File server present... ✔
Connecting to file server... ✘

File server reported username/password credentials invalid.
```

**Comment by shasheene** [comment]
File manager one-click mount does not refresh after restore:

On Rescuezilla v1.0.5, the one-click mounting using the file manager (PCManFM) does not work after a restore of a partition, with an error dialogue saying “No object for D-Bus interface”. Restarting the udisks service is insufficient to fix this behavior, but the easiest workaround is to open and close GParted, which seems to stop and start the correct things cause a refresh of the cache.

On first glance, Rescuezilla should have identical behavior on launch and exit as GParted. The task is to understand exactly what GParted is doing and why (it may turn out to be stopping other services too).

**Comment by shasheene** [comment]
The user mrtootoo posted a usability suggestion as part of his feedback on issue #15

> Once the backup was complete, although my drives were mounted, I did not have permission to access my drives. With v.104 I typically move my backup directory after completion of the back up process.

I wrote

> Rescuezilla v1.0.5 no longer runs the file manager as root. The logic was there is a big risk of a user accidentally deleting their own backup, or a critical system file. Perhaps the safest approach is to improve the Rescuezilla's destination directory selection to make it not necessary to manually move the backup directory.

Given the file manager runs under the ubuntu user, and Rescuezilla runs with elevated permissions (as root) it does make some sense to write the files as root. It equally makes sense to write the files as the initial user which launched Rescuezilla (ubuntu user). If the enduser has a Linux environment, the ubuntu UID/GID may not exist on their target system (or may be another existing user) in which case writing it as root user (UID/GID 1) is still fine. Currently I think writing it as the initial user who launched Rescuezilla is ideal as it will the best for most users when Rescuezilla is available from package managers across many Linux distributions.

**Comment by shasheene** [comment]
Users have [accidentally selected](https://sourceforge.net/p/redobackup/discussion/1169664/thread/903fac7d34/) the source drive to be the Rescuezilla boot media and seen cryptic errors. Due to how Rescuezilla processes device nodes, the cryptic error displayed is different whether the boot media is USB stick or a CD-ROM.

Given that backing up of CD-ROM drives (and floppy disks) isn't even supported, these device nodes  (and Rescuezilla mount points) should not be shown the enduser. This issue can be improved with #29 and #49.

**Comment by shasheene** [comment]
Most of these items have been fixed in [Rescuezilla v2.0](https://github.com/rescuezilla/rescuezilla/releases/latest).

Things like support for imaging CDs and network share improvements have not been done, but if required new issues can be created as relevant.
